### PR TITLE
support underlay mode with single nic

### PR DESCRIPTION
**Pull request by zhangzujian**
#### What type of this PR
 enhancement

### CODE REVIEW

**Comment by zhangzujian** [comment]
@oilbeater please review

File: pkg/daemon/ovs.go [hunk 1]
```diff
@@ -495,11 +499,82 @@ func providerBridgeExists() (bool, error) {
...
+	if err != nil {
+		return fmt.Errorf("failed to get sysctl %s: %v", sysctlDisableIPv6, err)
+	}
+	if disableIPv6 != "0" {
+		if _, err = sysctl.Sysctl(sysctlDisableIPv6, "0"); err != nil {
+			return fmt.Errorf("failed to enable ipv6 on OVS bridge %s: %v", brName, err)
+		}
+	}
+
+	addrs, err := netlink.AddrList(nic, unix.AF_UNSPEC)
```
Comment (lut777) [inline pkg/daemon/ovs.go:525]: addrs, err := netlink.AddrList(link, netlink.FAMILY_ALL)
FAMILY_ALL is easier to read I think

File: pkg/daemon/ovs.go [hunk 2]
```diff
@@ -495,11 +499,82 @@ func providerBridgeExists() (bool, error) {
...
+	if err != nil {
+		return fmt.Errorf("failed to get addresses on nic %s: %v", nicName, err)
+	}
+	routes, err := netlink.RouteList(nic, unix.AF_UNSPEC)
```
Comment (lut777) [inline pkg/daemon/ovs.go:529]: Same

**Review by lut777 — CHANGES_REQUESTED** [review]

File: pkg/daemon/ovs.go (hunk 1 above)
Comment (zhangzujian) [reply pkg/daemon/ovs.go]: > addrs, err := netlink.AddrList(link, netlink.FAMILY_ALL)
> FAMILY_ALL is easier to read I think

fixed

File: pkg/daemon/ovs.go (hunk 2 above)
Comment (zhangzujian) [reply pkg/daemon/ovs.go]: > Same

fixed

**Review by lut777 — APPROVED** [review]

**Pull request merged**